src/@carbon-info/views/IntroPage/IntroPage.tsx

```diff
@@ -42,6 +43,10 @@ const IntroPage: React.FC = () => {
     window.scrollTo(0, height);
   };
 
+  const goToAltCarbon = () => {
+    window.scrollTo(0, 10000);
```

Comment:
Instead of adding an absolute number, think it will be good to use `document.documentElement.getBoundingClientRect().height` instead
i.e.
```
window.scrollTo({ left: 0, top: document.documentElement.getBoundingClientRect().height ?? document.body.scrollHeight, behavior: "smooth" });
```